shared_libraries/orca_shared/recovery/shared_recovery.py

```diff
@@ -191,18 +191,30 @@ def post_entry_to_queue(
     md5_body = hashlib.md5(body.encode("utf8")).hexdigest()  # nosec
 
     LOGGER.debug("Sending message to the QUEUE")
-    response = mysqs.send_message(
-        QueueUrl=db_queue_url,
-        MessageDeduplicationId=deduplication_id,
```

Comment:
Just to be clear, we're fine with losing deduplication? Does something else take care of that?

Reply:
deduplication is only required for FIFO queue. since this now a standard queue, that is not needed.